plugins/inputs/exec/sample.conf

```diff
@@ -28,3 +28,7 @@
   ## them here:
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
   # data_format = "json"
+
+  ## Run the exec plugin in debug mode
+  ## If set to true, exec will not truncate the command's stderr output.
+  # debug = false
```

Comment:
Please do not introduce a new variable! Instead reuse the log-level of the `telegraf.Logger` which can be accessed by `Log.Level()`.

Reply:
Done. Changed the code to get the log level when Init() runs and set `debug` to true when Trace is set.